postgres/sql/create_tables.sql

```diff
@@ -50,8 +50,9 @@ CREATE TABLE IF NOT EXISTS Employee (
 
 CREATE TABLE IF NOT EXISTS CompanyHistory (
   company_id varchar(250) REFERENCES Company(company_id) ON UPDATE CASCADE ON DELETE CASCADE,
-  time_fetched INT NOT NULL,
-  trading_price MONEY NOT NULL
+  time_fetched DECIMAL NOT NULL,
+  trading_price MONEY NOT NULL,
+  num_shares INT NOT NULL
```

Comment:
Company history is only keeping track of historical share prices. It should not include the number of shares we have

Reply:
Another problem, this will break the current script and is more work

Reply:
Done